ec/acc/testdata/extension_bundle_download.tf

```
@@ -0,0 +1,6 @@
+resource "ec_extension" "my_extension" {
+  name           = "%s"
+  version        = "*"
+  extension_type = "bundle"
```

This test will download a `plugin` not a bundle, needs to be changed:
```suggestion
  extension_type = "plugin"
```

---

fixed by d340fa3